File: API-MS_28jun2023/HENRIQUE FERREIRA PAIVA/Prova - Henrique Ferreira Paiva/saudacao.py
def main():
    open('./alegria.txt', encoding='UTF-8')
    nome = str(input('Digite o seu nome: '))
    digito = -1


def fazer_saudacao(digito):
    if (digito > 0):
        print('Olá, ', end='')
    else:
        print('Tchau, ', end='')


main()

File: API-MS_28jun2023/GABRIEL DA SILVA LEAL/prova/saudacao.js
import {readFileSync} from 'fs'
import prompt from 'prompt-sync'
let input = prompt()

function main(){

    let numero = 1

    while(numero > 0){
    numero = Number(input('Digite um número, se for > 0 o programa continuará, se for menor ou igual, será encerrado!  '))
    let nome = input('Digite seu nome: ')
    let texto = readFileSync('./alegria.txt', 'utf8')
    let frase = texto.replaceAll("\r", "").split('\n')
    let linha_da_frase = sortear_frase(frase)

    if(numero > 0){
    console.log(`Olá ${nome} ${frase[linha_da_frase]}`)

    }

    if(numero <= 0){

        console.log(`Tchau! ${nome} ${frase[linha_da_frase]}`)

        break
    }
    }

    
}
main()

function sortear_frase(frase){

    let posicao = Math.floor(Math.random() * frase.length)

    return posicao
}